server/svix-server/src/webhook.rs

```diff
@@ -0,0 +1,298 @@
+// SPDX-FileCopyrightText: © 2022 Svix Authors
+// SPDX-License-Identifier: MIT
+
+use chrono::Utc;
+use http::{HeaderMap, HeaderValue};
+
+#[derive(thiserror::Error, Debug)]
```

Comment:
This structure is going to be exposed in our rust library. I don't know whether using `thiserror` is a good idea in a library or not. Let's keep it this way for now unless you or @svix-daniel have any thoughts.

Reply:
`thiserror` is fine for libs in my opinion.

Something to note is that It's Rust convention to have errors in lowercase (minus proper nouns) so that it plays well with others in the `Debug` display.